From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Thu, 16 Jul 2020 22:33:01 +0900
Subject: [PATCH] FindRuby cygport

---
 Modules/FindRuby.cmake | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/Modules/FindRuby.cmake b/Modules/FindRuby.cmake
index 1bdee60336..42a3daf774 100644
--- a/Modules/FindRuby.cmake
+++ b/Modules/FindRuby.cmake
@@ -409,9 +409,10 @@ if( Ruby_FIND_VERSION VERSION_GREATER_EQUAL "1.9"  OR  Ruby_VERSION VERSION_GREA
   find_path(Ruby_CONFIG_INCLUDE_DIR
     NAMES ruby/config.h  config.h
     HINTS
-      ${Ruby_HDR_DIR}/${Ruby_ARCH}
-      ${Ruby_ARCH_DIR}
-      ${Ruby_ARCHHDR_DIR}
+      ${RUBY_HDR_DIR}/${RUBY_ARCH}
+      ${RUBY_HDR_DIR}
+      ${RUBY_ARCH_DIR}
+      ${RUBY_ARCHHDR_DIR}
   )
 
   set(Ruby_INCLUDE_DIRS ${Ruby_INCLUDE_DIRS} ${Ruby_CONFIG_INCLUDE_DIR} )
-- 
2.27.0

