From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Thu, 16 Jul 2020 22:33:04 +0900
Subject: [PATCH] FindPerlLibs cygport

---
 Modules/FindPerlLibs.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Modules/FindPerlLibs.cmake b/Modules/FindPerlLibs.cmake
index 7e27f31880..b17a181104 100644
--- a/Modules/FindPerlLibs.cmake
+++ b/Modules/FindPerlLibs.cmake
@@ -108,6 +108,7 @@ if (PERL_EXECUTABLE)
   if (NOT PERL_POSSIBLE_LIBRARY_NAMES)
     set(PERL_POSSIBLE_LIBRARY_NAMES perl${PERL_VERSION_STRING} perl)
   endif()
+  set(PERL_POSSIBLE_LIBRARY_NAMES ${PERL_POSSIBLE_LIBRARY_NAMES} perl${PERL_VERSION_STRING} perl)
   if (CMAKE_SYSTEM_NAME MATCHES "MSYS|CYGWIN")
     # on MSYS and CYGWIN environments, current perl -V:libperl gives shared library name
     # rather than the import library. So, extends possible library names
-- 
2.27.0

