From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lemures Lemniscati <lemures.lemniscati@gmail.com>
Date: Tue, 14 Jul 2020 22:33:07 +0900
Subject: [PATCH] FindTclsh cygport

---
 Modules/FindTclsh.cmake | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/Modules/FindTclsh.cmake b/Modules/FindTclsh.cmake
index e3bd11040b..7b7a685991 100644
--- a/Modules/FindTclsh.cmake
+++ b/Modules/FindTclsh.cmake
@@ -16,13 +16,6 @@ library is.  This code sets the following variables:
   TCLSH_FOUND = TRUE if tclsh has been found
   TCL_TCLSH = the path to the tclsh executable
 
-In cygwin, look for the cygwin version first.  Don't look for it later
-to avoid finding the cygwin version on a Win32 build.
-#]=======================================================================]
-
-if(CYGWIN)
-  find_program(TCL_TCLSH NAMES cygtclsh83 cygtclsh80)
-endif()
 
 get_filename_component(TK_WISH_PATH "${TK_WISH}" PATH)
 get_filename_component(TK_WISH_PATH_PARENT "${TK_WISH_PATH}" PATH)
-- 
2.27.0

